io/io/src/TMemFile.cxx

```diff
@@ -177,9 +177,6 @@ TMemFile::TMemFile(const char *path, char *buffer, Long64_t size, Option_t *opti
    EMode optmode = ParseOption(option);
 
    if (NeedsToWrite(optmode)) {
-      Int_t mode = O_RDWR | O_CREAT;
-      if (optmode == EMode::kRecreate) mode |= O_TRUNC;
-
```

Comment:
@pcanal it appears this variable is unused since the initial commit 88add5a1efb48d9de5b185198439edef56568d86. I thought about passing `mode` to `TMemFile::SysOpen`, but considering that it's been this way since 2011 I decided not to.

Reply:
Indeed, for consistency, it need to be passed to `SysOpen` but also, at least for now, `TMemFile::SysOpen` ignores the argument.